### DISCUSSION TITLE
Include additional command-line options for applying transforms

### DISCUSSION THREAD

**Discussion by hidadsoup** [microsoft/winget-cli#2910] [Ideas]
One thing preventing me from installing something like Adobe Reader via winget is that I cannot apply the transforms that I include with the traditional installation method. We prefer to disable all the "upsell" options in Adobe so users aren't presented with links to purchase functionality that isn't included in the free version. We also turn off things like desktop shortcut creation. This does present a challenge, however, because winget would need access to the MST file that includes the configurations.

**Comment by denelon** [comment]
It may be possible to use the "--override" argument to pass instructions to the installer. We don't have support for using "configuration files" with installers yet.

Somewhat related to:
* https://github.com/microsoft/winget-cli/issues/158

**Reply by cgerke** [reply]
It is possible using a transform.

Download the Adobe Customisation Wizard https://helpx.adobe.com/creative-cloud/kb/deploying-acrobat-singleapp.html#:~:text=After%20installation%20of%20Customization%20wizard%2C%20launch%20the%20wizard,ready%20to%20be%20deployed%20on%20the%20users%20machines.

--override "/sAll /rs /l /msi /qn TRANSFORMS=""$pwd\Files\transform.mst"""

Although depending on how often you may change settings you may want to manage the Adobe policies via GPO or ingesting to your preferred MDM.

https://www.adobe.com/devnet-docs/acrobatetk/tools/DesktopDeployment/gpo.html#gpo-registry-template
